source/_components/cover.lcn.markdown

```diff
@@ -0,0 +1,22 @@
+---
+layout: page
+title: "LCN Cover"
+description: "Instructions on how to setup LCN covers within Home Assistant."
+date: 2018-11-01 08:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: lcn.png
+ha_category: Cover
+ha_release: 0.89
+ha_iot_class: "Local Push"
+---
```

Comment:
We're merging platform pages that don't need details with the component page. We've already done this for the existing LCN pages.

We can remove the cover page.